observer/probers/tls/tls.go

```diff
@@ -0,0 +1,167 @@
+package probers
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"github.com/prometheus/client_golang/prometheus"
+	"golang.org/x/crypto/ocsp"
+	"io"
+	"net/http"
+	"time"
+)
+
+const (
+	Success = iota
```

Comment:
As discussed on the call: since we're going to be using this `iota` for errors, then our success case will be `nil`. You can go ahead and remove `Success`.

Also, these should all be lower-case since we aren't exporting them.

```suggestion
	Success = iota
```